pkg/scheduler/core/equivalence/eqivalence.go

```diff
@@ -300,6 +347,11 @@ func (n *NodeCache) invalidatePreds(predicateKeys sets.String) {
 	defer n.mu.Unlock()
 	for predicateKey := range predicateKeys {
 		delete(n.cache, predicateKey)
+		if _, ok := n.generations[predicateKey]; ok {
```

Comment:
Similar to above, you can rely on the default map behavior here.
e.g.

```go
m := make(map[string]int)
fmt.Println(m) //  "map[]"
m["counter"]++
fmt.Println(m) // "map[counter:1]"
```

Reply:
Fixed.